scripts/quests/windurst/RNG_AF2_Fire_and_Brimstone.lua

```diff
@@ -0,0 +1,173 @@
+-----------------------------------
+-- Fire and Brimstone (RNG AF2)
+-----------------------------------
+-- Log ID: 2, Quest ID: 73
+-- Perih Vashai: !pos 117 -3 92 241
+-- Koh Lenbalalako: !pos -64.412 -17 29.213 249
+-- Gravestone: !pos 180 -32 -56 104
+-- qm2: !pos -10.946 -1.000 313.810 104
+-----------------------------------
+require('scripts/globals/items')
+require("scripts/globals/keyitems")
+require('scripts/globals/quests')
+require('scripts/globals/status')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.FIRE_AND_BRIMSTONE)
+
+quest.reward =
+{
+    item     = xi.items.HUNTERS_BERET,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:getMainLvl() >= xi.settings.main.AF2_QUEST_LEVEL and
+            player:getMainJob() == xi.job.RNG and
+            player:hasCompletedQuest(xi.quest.log_id.WINDURST, xi.quest.id.windurst.SIN_HUNTING)
+        end,
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            ['Perih_Vashai'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(531)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [531] = function(player, csid, option, npc)
+                    quest:begin(player)
+                    quest:setVar(player, 'Prog', 1)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            ['Perih_Vashai'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') > 0 and quest:getVar(player, 'Prog') < 4 then
+                        return quest:event(532) -- during RNG AF2
+                    elseif quest:getVar(player, 'Prog') == 4 then
+                        return quest:progressEvent(535, 0, 13360, xi.items.OLD_EARRING)
```

Comment:
does 13360 reference the twinstone earring? if so we could put the items reference here, and anywhere else 13360 is in dialogue